dialogue-core/src/main/java/com/palantir/dialogue/core/DialogueChannel.java

```diff
@@ -110,6 +113,9 @@ public void updateUris(Collection<String> uris) {
                 ImmutableList.copyOf(limitedChannelByUri.values()),
                 random,
                 channelName));
+
+        // some stuck requests might be able to make progress now
+        queuedChannel.schedule();
```

Comment:
This change also means that if we had n fully saturated nodes (maybe stuck processing very slow requests) and another node comes online, that new node will start receiving requests from the queue.

Before this PR, the new node would not be utilized until one of our existing n nodes returned something (or a new request came in).

Reply:
it seems pretty strange to me that this channel would know about the QueuedChannel at all, and this is why the original queued channel had that scheduled poke every 100ms.

Reply:
I've actually just pulled this change out to a separate PR so it doesn't get too conflated from the zero URI behaviour (which is gonna change anyway): https://github.com/palantir/dialogue/pull/605 Shall we discuss there?